browser/help/topics/v3-ht-annotations.md

```diff
@@ -159,17 +180,14 @@ The gnomAD v3 Hail Table annotations are defined below:
-- **splice_ai**:
-  - **splice_ai**: The maximum delta score, interpreted as the probability of the variant being splice-altering.
-  - **splice_consequence**: The consequence term associated with the max delta score in 'splice_ai_max_ds'.
-  - **has_duplicate**: Whether the variant has more than one splice_ai score associated with it.
-- **primate_ai**:
-  - **primate_ai_score**: PrimateAI's deleteriousness score from 0 (less deleterious) to 1 (more deleterious).
-  - **has_duplicate**: Whether the variant has more than one primate_ai_score associated with it.
+- **in_silico_predictors**: Variant prediction annotations. Struct contains prediction scores from multiple in silico predictors for variants that are predicted to be missense, impacting protein function, evolutionarily conserved, or splice-altering. We chose scores for either MANE Select or canonical transcripts if a prediction score was available for multiple transcripts.
+  - **revel_max**: Rare Exome Variant Ensemble Learner (REVEL) [score](https://www.cell.com/ajhg/fulltext/S0002-9297(16)30370-6). An ensemble score for predicting the pathogenicity of missense variants (based on 13 other variant predictors). Score predicting the pathogenicity of missense variants with a range of 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+  - **primateai3d_max**: A deep neural network-trained [score](https://www.science.org/doi/10.1126/science.abn8197) for classifying pathogenicity of missense variants. Score ranging from 0 to 1 predicting the deleteriousness of missense variants. Higher scores indicate a variant is more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
+  - **cadd**: Combined Annotation Dependent Deletion [score](https://academic.oup.com/nar/article/47/D1/D886/5146191) used to predict deleteriousness of SNVs and indels.
+    - **phred**: CADD Phred-like scaled C-scores ranging from 1 to 99 based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Larger values indicate increased predicted deleteriousness.
+    - **raw_score**: Unscaled CADD scores indicating whether a variant is likely to be "observed" (negative values) vs "simulated" (positive values). Larger values indicate increased predicted deleteriousness.
+  - **sift_max**: Sorting Tolerant From Intolerant (SIFT) [score](https://www.nature.com/articles/nprot.2009.86) used to predict the impact of SNVs on protein function. Scaled probability of the amino acid substitution being tolerated, ranging from 0 to 1. Scores below 0.05 are predicted to impact protein function. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
```

Comment:
```suggestion
  - **sift_max**: [Score](https://www.nature.com/articles/nprot.2009.86) reflecting the scaled probability of the amino acid substitution being tolerated, ranging from 0 to 1. Scores below 0.05 are predicted to impact protein function. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
```